docs/docs/common-api-reference.md

```
@@ -0,0 +1,52 @@
+id: common-api-reference
+title: API Reference Overview
+sidebar_position: 6
+---
+
+# API Reference Overview
+
+Img2Num provides bindings for multiple languages. Choose the one that fits your workflow:
+
+| Language              | Docs                                                      |
+| --------------------: | :------------ |
+| <a href="https://github.com/Ryan-Millard/Img2Num/releases?q=bindings-c"><img src="https://cdn.jsdelivr.net/gh/devicons/devicon@latest/icons/c/c-original.svg" width="30" /></a> | [JS API Reference](/docs/js/js-api-reference)             |
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)